client/src/components/Notifications/Broadcasts/BroadcastsOverlay.test.ts

```diff
@@ -68,19 +70,19 @@ describe("BroadcastsOverlay.vue", () => {
 
     it("should render only one broadcast at a time", async () => {
         const wrapper = await mountBroadcastsOverlayWith(FAKE_BROADCASTS);
-        expect(wrapper.findAll(".broadcast-message")).toHaveLength(1);
-        expect(wrapper.find(".broadcast-message").text()).toContain("Test message 1");
+        expect(wrapper.findAll(messageCssSelector)).toHaveLength(1);
+        expect(wrapper.find(messageCssSelector).text()).toContain("Test message 1");
     });
 
     it("should render the next broadcast when the current one is dismissed", async () => {
         const wrapper = await mountBroadcastsOverlayWith(FAKE_BROADCASTS);
-        expect(wrapper.findAll(".broadcast-message")).toHaveLength(1);
-        expect(wrapper.find(".broadcast-message").text()).toContain("Test message 1");
+        expect(wrapper.findAll(messageCssSelector)).toHaveLength(1);
+        expect(wrapper.find(messageCssSelector).text()).toContain("Test message 1");
 
-        const dismissButton = wrapper.find("#dismiss-button");
+        const dismissButton = wrapper.find(".dismiss-button");
         await dismissButton.trigger("click");
 
-        expect(wrapper.findAll(".broadcast-message")).toHaveLength(1);
-        expect(wrapper.find(".broadcast-message").text()).toContain("Test message 2");
+        expect(wrapper.findAll(messageCssSelector)).toHaveLength(1);
+        expect(wrapper.find(messageCssSelector).text()).toContain("Test message 2");
     });
 });
```

Comment:
Since you've added in the fuctionality to view multiple messages in a carousel, maybe we should add a test case for that?

Reply:
Good call! I'll add a test case